Adding documentation for Handle case of table with no supported columns (#21472)
prestodb/presto: presto-docs/src/main/sphinx/connector/postgresql.rst

If you used a different name for your catalog properties file, use
that catalog name instead of ``postgresql`` in the above examples.

Tables with Unsupported Columns
-------------------------------

> When you query a PostgreSQL table via the Presto connector, and the table either has no supported columns or contains

Reviewer: ```suggestion
If you query a PostgreSQL table with the Presto connector, and the table either has no supported columns or contains
```
Avoid Latin abbreviations. See the [GitLab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via). I made one suggestion here. See the entry for via in the link for other phrasing you could use.

> only unsupported data types, Presto will return an error. The error message will explicitly state the problem with the

Reviewer: ```suggestion
only unsupported data types, Presto will return an error. The error message will be similar to the following example:
```
You described the error message in #21422 . Consider adding the error message here for the user to read.

> table's columns.

Reviewer: ```suggestion
```

If you used a different name for your catalog properties file, use
that catalog name instead of ``postgresql`` in the above examples.

Tables with Unsupported Columns
-------------------------------

If you query a PostgreSQL table with the Presto connector, and the table either has no supported columns or contains

> only unsupported data types, Presto will return an error. The error message will be similar to the following example:

Reviewer: ```suggestion
only unsupported data types, Presto returns an error similar to the following example:
```
Suggestion as shorter and simpler, and to avoid future tense in documentation.

> Query 20231120_102910_00004_35dqb failed: Table 'public.unsupported_type_table' has no supported columns (all 1 columns are not supported).

Reviewer: ```suggestion
``Query 20231120_102910_00004_35dqb failed: Table 'public.unsupported_type_table' has no supported columns (all 1 columns are not supported).``
```
Suggestion to mark the error message as different from normal text.
